test/unit/test_services/test_kubectl.py

```diff
@@ -48,7 +48,13 @@ def it_returns_output(mocker):
         obj = Kubectl('staging')
         mock_return = { 'foo': 'bar' }
         mocker.patch('hokusai.services.kubectl.shout', return_value=json.dumps(mock_return))
+        spy = mocker.spy(hokusai.services.kubectl, 'shout')
         assert obj.get_object('pods') == mock_return
+        spy.assert_has_calls([
+          mocker.call(
+            '/tmp/.local/bin/kubectl --context staging get pods -o json'
+          )
+        ])
```

Comment:
ha. nice. wonder why i didn't do this last time. likely b/c i felt it was too mundane to spy.